nemo/collections/vlm/neva/data/sequence_packing.py

```diff
@@ -93,6 +93,13 @@ def predict_seq_len(instance_tokens: torch.Tensor, num_image_embeddings_per_tile
     return seqlen
 
 
+def predict_seq_len_with_padding(instance_tokens: torch.Tensor, pad_to_multiple_of: int = 64) -> int:
```

Comment:
can we merge this with `predict_seq_len`? or it's for a different input use case?

Reply:
Hi yes, They are completely different `predict_seq_len` is used in Neva whereas `predict_seq_len_with_padding` in llava-next. We can merge them moving forward ig.